Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Entries/RootAssetBrowserEntry.cpp

```diff
@@ -391,6 +393,40 @@ namespace AzToolsFramework
                 return parent;
             }
 
+            // If the project is not inside the root engine folder
+            if (!m_projectPath.IsRelativeTo(m_enginePath))
+            {
+                // Update the parent to be the project directory
+                AZ::IO::FixedMaxPath directory;
+                if (absolutePathView.IsRelativeTo(m_projectPath) && !inProjectDirectory)
+                {
+                    for (auto pathIter = m_projectPath.begin(); pathIter != m_projectPath.end(); ++pathIter)
+                    {
+                        if (std::next(pathIter) == m_projectPath.end())
+                        {
+                            break;
+                        }
+                        directory = directory / *pathIter;
+                    }
+                    parent->m_fullPath = directory;
+                    inProjectDirectory = true;
+                }
+                // Update the parent to be the o3de directory
+                else if (absolutePathView.IsRelativeTo(m_enginePath) && inProjectDirectory)
+                {
+                    for (auto pathIter = m_enginePath.begin(); pathIter != m_enginePath.end(); ++pathIter)
+                    {
+                        if (std::next(pathIter) == m_enginePath.end())
+                        {
+                            break;
+                        }
+                        directory = directory / *pathIter;
+                    }
+                    parent->m_fullPath = directory;
```

Comment:
Same here with the enginePath directory.

All this is doing is copying the m_enginePath variable to parent->m_fullPath
```suggestion
                parent->m_fullPath = m_enginePath;
```

Reply:
Same as above, this is gets the path leading up to the engine so that we can create the 'o3de' folder as a separate folder

Reply:
repeat: The AZ::IO::ParentPath function should be used
https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/IO/Path/Path.h#L202-L206

Reply:
Thanks, I'll update this